tools/gradle-plugin/build.gradle.kts

```diff
@@ -46,4 +46,8 @@ tasks.named<Test>("test") {
     useJUnitPlatform()
 }
 
+tasks.named("pluginUnderTestMetadata") {
+    dependsOn("processJandexIndex")
```

Comment:
Not really related to this PR, fixes this build warning:
```
> Task :pluginUnderTestMetadata
Execution optimizations have been disabled for task ':pluginUnderTestMetadata' to ensure correctness due to the following reasons:
  - Gradle detected a problem with the following location: '/home/snazy/devel/misc/smallrye-open-api/tools/gradle-plugin/build/resources/main'. Reason: Task ':pluginUnderTestMetadata' uses this output of task ':processJandexIndex' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed. Please refer to https://docs.gradle.org/7.6/userguide/validation_problems.html#implicit_dependency for more details about this problem.
```